container_pipeline/utils.py

```diff
@@ -1,10 +1,14 @@
+import hashlib
 import os
 
 
 def get_job_name(job_details):
     """Get jenkins job name from job_detials"""
-    return str(job_details['appid']) + "-" + str(job_details['jobid']) + "-" \
-        + str(job_details['desired_tag'])
+    namespace = str(job_details['appid']) + "-" + str(job_details['jobid']) + \
+        "-" + str(job_details['desired_tag'])
+    project = hashlib.sha224(namespace).hexdigest()
+
+    return project, namespace
```

Comment:
@navidshaikh why do you find this complicated? I can change the name, docstring, etc to make things clear. But I can't simplify this any further. And there are places where we're interested in namepsace+hash. This function does precisely that. Why should we write a different function then?